va/va.h

```diff
@@ -86,13 +86,22 @@
 extern "C" {
 #endif
 
-#if defined(__GNUC__) && !defined(__COVERITY__)
+#if defined(__cplusplus) && __cplusplus >= 201402L
+#define va_deprecated [[deprecated]]
+#define va_deprecated_enum [[deprecated]]
+#elif __STDC__ && __STDC_VERSION__ >= 202311L
+#define va_deprecated [[deprecated]]
+#define va_deprecated_enum [[deprecated]]
```

Comment:
Don't know how much we care, but this will break Coverity Scan - both within this project and other projects that use libva. See 392187428cee9f5062c1b989cbe83a0a5e17c861 for details.

Reply:
Added notation is a part of C/C++ standard which tools should now or later be capable to understand. Whether Coverity understands above notation as of today - that's a question. I wonder how we can check before the merge? @uartie : do we have a way?

Other than that it's easy to modify PR to expand to nothing for coverity (as it was before).